### ISSUE TITLE
Add snapcraft navigation to discourse

### ISSUE THREAD

**Issue by solazio** [canonical-web-and-design/snapcraft.io#3002]

**Comment by cristinadresch** [comment]
@anthonydillon what about the change in the other websites? should we wait to implement on snapcraft too, and then roll out the sticky change to other sites later?

**Comment by anthonydillon** [comment]
Yeah exactly, once @solazio has the solution copying it to the other sites will be an easy task.

**Comment by solazio** [comment]
@tbille what was the outcome of the meeting you had on this?
